app/components/Contacts/ContactModal.js

```diff
@@ -47,7 +46,9 @@ const ContactModal = ({
   }
 
   // the remote node for the channel
-  const node = find(channelNodes, { pub_key: channel.remote_pubkey })
+  const node = channelNodes.find(channel => {
+    channel.remote_pubkey
+  })
```

Comment:
The new code doesn't replicate the earlier behavior. Here's what would work AFAICT:
```js
const node = channelNodes.find(node => (
  node.pubkey == channel.remote_pubkey
))
```
Note `{}` functions require explicit return, `()` returns the value of the expression.